src/process.cpp

```diff
@@ -150,12 +150,14 @@ int proc_t::execute(int app_id) {
                                             find_working_directory(cmd, _env) :
                                             boost::filesystem::path(proc.working_dir);
     BOOST_LOG(info) << "Spawning ["sv << cmd << "] in ["sv << working_dir << ']';
-    auto child = platf::run_unprivileged(cmd, working_dir, _env, _pipe.get(), ec);
+    bp::group child_group;
```

Comment:
Note: I don't really like this myself that there is a need to pass a temporary group, but I don't know how to do it optionally without creating 2 function versions. The `boost::child` constructor is just meh...